prow/cmd/deck/main.go

```diff
@@ -83,36 +116,36 @@ func main() {
 	}
 
 	// locally just serve from ./static, otherwise do the full main
-	if *runLocal {
+	if o.runLocal {
 		mux.Handle("/", staticHandlerFromDir("./static"))
 	} else {
 		mux.Handle("/", staticHandlerFromDir("/static"))
-		prodOnlyMain(mux)
+		o.prodOnlyMain(mux)
 	}
 
 	// setup done, actually start the server
 	logrus.WithError(http.ListenAndServe(":8080", mux)).Fatal("ListenAndServe returned.")
 }
 
 // prodOnlyMain contains logic only used when running deployed, not locally
-func prodOnlyMain(mux *http.ServeMux) {
+func (o *options) prodOnlyMain(mux *http.ServeMux) {
```

Comment:
er should this be on options or take in an options?

Reply:
FWIW this whole method is a bit gross and mostly my fault.

Reply:
Didn't really feel strongly either way, moved it to a parameter